appveyor.yml

```diff
@@ -0,0 +1,26 @@
+environment:
+  nodejs_version: "6"
+  EXTENSION_ID:
+    secure: 1sDEVQM+HCPKqyAwafg5BUi7d+fVcyoYuzxuBRGfsRyju2QC7YVy9ha9Ze6aNZMZ
```

Comment:
Probably don't want these in code.  In you project in Appveyor, go to the settings (⚙️ to the right), then Environment, and you can add them there.

Reply:
this is a secured version i need them as env variable in the npm. secure value should be decrypted only for pushpay-labs org member pr (following the configuration on screenshot)

Reply:
I don't know which is better (env vars or secure vars) - but Erwan's usage of secure variables seems right here according to [appveyor docs](https://www.appveyor.com/docs/build-configuration/#secure-variables). These values are encrypted already.

Reply:
How does appveyor decrypt it? Do you store a key anywhere?

Reply:
Appveyor generates a key for each account